src/agent/onefuzz-agent/src/tasks/merge/libfuzzer_merge.rs

```diff
@@ -162,6 +163,7 @@ pub async fn merge_inputs(
         &config.target_exe,
         &config.target_options,
         &config.target_env,
+        config.common.setup_dir.clone(),
```

Comment:
Why does this need to clone?

Reply:
these clone were there because i had changed the type of the parameter. I reverted those changes